roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -107,30 +107,27 @@ def get_var(self, *keys, **kwargs):
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
 
-        convert = kwargs.get("type")
+        convert = kwargs.get("convert", lambda x: x)
         try:
-            if convert is None:
-                pass
-            elif convert in ["str", str]:
-                return str(value)
-            elif convert in ["int", int]:
-                return int(value)
-            elif convert in ["float", float]:
-                return float(value)
-            elif convert in ["bool", bool]:
+            if convert is bool:
```

Comment:
I really doubt there is an actual use case for using the built-in `bool` as the converter. In this context it is essentially `non_empty` so calling it `bool` would be highly confusing and lead to bugs.